src/usermod.c

```diff
@@ -1952,7 +1952,11 @@ static void update_lastlog (void)
 		}
 	}
 
-	(void) close (fd);
+	if (close (fd) != 0 && errno != EINTR) {
+		fprintf (stderr,
+		         _("%s: failed to copy the lastlog entry of user %lu to user %lu: %s\n"),
+		         Prog, (unsigned long) user_id, (unsigned long) user_newid, strerror (errno));
```

Comment:
`[u]intmax_t` and `%jd` or `%ju` is shorter.  Now that we require C11, we could use it.